desktop_version/src/Render.cpp

```diff
@@ -338,6 +343,16 @@ void menurender()
         }
 
 
+        break;
+    case Menu::language:
+        if (loc::languagelist.size() == 0)
```

Comment:
Can be changed to `loc::languagelist.empty()`. Doesn't really make a difference functionally, but it makes the code a bit clearer.

Reply:
Changed in 57608cdffdf1656cd93c73370784c66a502aed43.